### PR TITLE
Add ctrl-p ctrl-n keybind

### PR DESCRIPTION
**Pull request by takeokunn**
I added keybind to make it more comfortable.

Thanks!

### CODE REVIEW

**Comment by takeokunn** [comment]
README.md更新しました
こんな感じでどうでしょうか?

File: README.md [hunk 1]
```diff
@@ -86,60 +86,60 @@ $ bash build.sh
...
+| volume list      | remove volume          | <kbd>d</kbd>                                                   |
+| volume list      | prune volume           | <kbd>p</kbd>                                                   |
+| volume list      | inspect volume         | <kbd>Enter</kbd> / <kbd>o</kbd>                                |
+| volume list      | refresh volume list    | <kbd>Ctrl</kbd> + <kbd>r</kbd>                                 |
+| volume list      | filter image           | <kbd>f</kbd>                                                   |
+| network list     | inspect network        | <kbd>Enter</kbd> / <kbd>o</kbd>                                |
+| network list     | remove network         | <kbd>d</kbd>                                                   |
+| pull image       | pull image             | <kbd>Enter</kbd>                                               |
+| pull image       | close panel            | <kbd>Esc</kbd>                                                 |
+| create container | next input box         | <kbd>↓</kbd> / <kbd>Tab</kbd> / <kbd>Ctrl</kbd> + <kbd>n</kbd> |
```
Comment (skanehira) [inline README.md]: create container panel cannot work in this PR.
If you want use the key bind, please add key bind on create container panel.

https://github.com/skanehira/docui/blob/master/panel/form.go#L42

File: README.md [hunk 2]
```diff
@@ -86,60 +86,60 @@ $ bash build.sh
...
+| create container | previous input box     | <kbd>↑</kbd> / <kbd>Ctrl</kbd> + <kbd>p</kbd>                  |
+| create container | close panel            | <kbd>Esc</kbd>                                                 |
+| detail           | cursor dwon            | <kbd>j</kbd>                                                   |
+| detail           | cursor up              | <kbd>k</kbd>                                                   |
+| detail           | page dwon              | <kbd>d</kbd>                                                   |
+| detail           | page up                | <kbd>u</kbd>                                                   |
+| search images    | search image           | <kbd>Enter</kbd>                                               |
+| search images    | close panel            | <kbd>Esc</kbd>                                                 |
+| images           | next image             | <kbd>j</kbd> / <kbd>↓</kbd> / <kbd>Ctrl</kbd> + <kbd>n</kbd>   |
```
Comment (skanehira) [inline README.md]: images panel cannot work in this PR.
If you wan to user the key bind, please add following place.

https://github.com/skanehira/docui/blob/master/panel/searchImageResultPanel.go#L81

File: README.md [hunk 3]
```diff
@@ -86,60 +86,60 @@ $ bash build.sh
...
+| images           | previous image         | <kbd>k</kbd> / <kbd>↑</kbd> / <kbd>Ctrl</kbd> + <kbd>p</kbd>   |
+| images           | pull image             | <kbd>Enter</kbd>                                               |
+| images           | close panel            | <kbd>Esc</kbd>                                                 |
+| create volume    | close panel            | <kbd>Esc</kbd>                                                 |
+| create volume    | next input box         | <kbd>↓</kbd> / <kbd>Tab</kbd> / <kbd>Ctrl</kbd> + <kbd>n</kbd> |
```
Comment (skanehira) [inline README.md]: https://github.com/skanehira/docui/pull/77/commits/22ed2caaffc0a349b9875ea6d4e1bfb6dd35c8b0#diff-04c6e90faac2675aa89e2176d2eec7d8R127

**Comment by skanehira** [comment]
レスがないので閉じます

**Pull request closed without merging**